src/iap/layout/components/layout-header/layout-header.component.scss

```
@@ -3,7 +3,7 @@
 
 //TODO: make hamburger menu
 
-.header {
+.iap-header {
```

:point_right: :point_left: 
```suggestion
.iap-layout-header {
```